src/App.tsx

```diff
@@ -133,9 +133,27 @@ const TMP_DATA: MarkerFileData = {
 	}]
 };
 
-export default function App() {
+    function App() {
```

Comment:
Sorry, when we were troubleshooting I didn't realize deleting the export keyword was from the App() function. You'll have to add `export default` back. I thought you might have been exporting your own local helper function.